services/QuillLMS/db/migrate/20230301160642_create_admin_approval_request.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateAdminApprovalRequest < ActiveRecord::Migration[6.1]
+  def change
+    create_table :admin_approval_requests do |t|
+      t.references :admin_info, foreign_key: true, dependent: :destroy
```

Comment:
I'm not familiar with `dependent: :destroy` in migration.  Is this a new convention?  Normally I put it in the model file.

Reply:
You know, I was looking at `https://github.com/empirical-org/Empirical-Core/blob/develop/services/quilllms/db/migrate/20230206203447_create_user_email_verification.rb#L6` as a reference and didn't really think about it, but further investigation indicates that it doesn't do anything in migrations so I think that was just a mistake. I've removed it and added it to the model file.